include/sparrow_ipc/serialize_primitive_array.hpp

```diff
@@ -50,18 +53,22 @@ namespace sparrow_ipc
     }
 
     template <typename T>
-    sparrow::primitive_array<T> deserialize_primitive_array(const std::vector<uint8_t>& buffer) {
+    sparrow::primitive_array<T> deserialize_primitive_array(const std::vector<uint8_t>& buffer)
```

Comment:
This should live in deserialize_primitive_array.hpp, or deserialize_primitive_array.hpp should be merged into this file.

Reply:
removed